.gitignore

```diff
@@ -1 +1,5 @@
+# Dependencies
 bower_components/
+
+# Editor / IDE Configuration
+.idea/
```

Comment:
Please don't add this to the project `.gitignore` -- this sounds like a local settings for you